### PR TITLE
Les 63

### PR DESCRIPTION
**Pull request by IgorTanov**
add linters and prettier with actual master branch (rework LES-63)

### CODE REVIEW

File: src/components/body/content/Main.tsx [hunk 1]
```diff
@@ -3,19 +3,19 @@ import Lessoner from "./lessoner/Lessoner";
...
+        <Route path="/users/sign_up" element={<FirstRegistrationForm />} />
         <Route path="/" element={<Lessoner />} />
         <Route path="/categories" element={<Categories />} />
         <Route path="/lessons" element={<Lessons />} />
         <Route path="/about" element={<About />} />
-        <Route path="/users/sign_in" element={<SignIn />} />
-        <Route path="/users/sign_up" element={<SignUp />} />
       </Routes>
     </div>
   );
```
Comment (Evgeniy121212) [inline src/components/body/content/Main.tsx:21]: please, not include that file into the PR

File: src/App.tsx [hunk 2]
```diff
@@ -1,18 +1,12 @@
 import "./App.css";
```
Comment (Evgeniy121212) [inline src/App.tsx:1]: please, not include that file into the PR

**Review by Evgeniy121212 — COMMENTED** [review]
please, remove lessoner_project_frontend from that PR

File: src/App.tsx (hunk 2 above)
Comment (IgorTanov) [reply src/App.tsx:1]: remove lessoner_project_frontend

File: src/components/body/content/Main.tsx (hunk 1 above)
Comment (IgorTanov) [reply src/components/body/content/Main.tsx:21]: remove this files

**Comment by Evgeniy121212** [comment]
Please, check is that actual PR or not

If not - let's close it

**Review by valer-sky — APPROVED** [review]

**Review by Evgeniy121212 — APPROVED** [review]

**Pull request closed without merging**